src/scripts/main.js

```diff
@@ -22,7 +22,22 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const item in data) {
+    const li = document.createElement('li');
+
+    li.textContent = item;
+    ul.append(li);
+
+    const innerObj = Object.keys(data[item]);
+
+    if (innerObj) {
```

Comment:
it would be better to check the `length`
```suggestion
    if (innerObj.length) {
```